### DISCUSSION TITLE
Wiki can't be attached to specific release or branch

### DISCUSSION THREAD

**Discussion by armleo** [community/community#11807] [Other Feature Feedback, Questions, & Ideas]
Let's say I wanted to use a Wiki to document an API for two different branches, but I couldn't, and here is the reason.

Let's say I have v2.0 and v3.0.
v3.0 is upstream and has API not compatible with v2.0.
v2.0 has an API that is documented in a Wiki.

There is no way to keep both Wikis.

For this, I suggest following change to Wiki.

As there is GitHub CI already located in .github, Wiki can be moved to .github/wiki. This will allow Wiki to be kept matching the current version.

This means that we can have a v3.0 (main) branch and v2.0 branch with two different wikis, while they don't interfere with each other.

For two branches this is no big deal, you can even keep two v2.0 and v3.0 in the same wiki. But for releases with multiple API changes per month, the current variant does not seem to be a reasonable way to do it. So keeping wiki in .github/wiki will allow branch-specific wiki and API documentation.

Right now we have a docs/ folder that contains all of the documentation for that version, this is good enough, but if we could merge "wiki" and docs it would be great.

**Comment by lsarrazi** [comment]
The real question is why isn't it already the case ?
Thats seems so obvious to me that I cant believe the github team didn't do that for a good reason though.
